.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Lint"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
IMO it is a waste to run a commit message validation/lint on every Node version. Do you expect cases when it would fail on one, but not the other? What do you think about making it a job running on the latest Node LTS in the `test.yml` instead?

Reply:
I think it should instead run in the oldest Node.js we support (i.e. 10), not on the latest one. I think it is safer to assume that if it works on an older version it will work on a newer one than vice versa. WDYT?

Reply:
It's a dev tool, I don't think we should care about old Node.js version, especially obsolete ones like Node.js 10.

Reply:
Let's get rid of Node.js 10 in https://github.com/karma-runner/karma-jasmine/pull/297. SG?

Reply:
@XhmikosR

Reply:
Any other thoughts before merging? @XhmikosR